crates/swc_common/src/syntax_pos.rs

```diff
@@ -237,6 +242,12 @@ impl Span {
     /// Returns `true` if this is a dummy span with any hygienic context.
     #[inline]
     pub fn is_dummy(self) -> bool {
+        self.lo.0 == 0 && self.hi.0 == 0 || self.lo.0 >= DUMMY_RESERVE
+    }
+
+    /// Returns `true` if this is a dummy span with any hygienic context.
+    #[inline]
+    pub fn is_dummy_with_cmt(self) -> bool {
```

Comment:
I think this name is the reverse of the actual impl?

How about `is_dummy_ignoring_cmt`?